### ISSUE TITLE
[Feature request] Add an anchor to comments

### ISSUE THREAD

**Issue by VincentTam** [robinmetral/eaudepoisson#36]
In many blog themes, when clicked on "# commentaire(s)" in the post header, the reader will be directed to the "commentaires".  It would be great to have such feature.

**Comment by robinmetral** [comment]
Yes! Do you want to submit a PR? :wink: Otherwise I'll work on it when I get the time
